src/firebase/index.js

```diff
@@ -0,0 +1,15 @@
+export const listenToMessages = (firestore) =>
+  firestore().collection("generalMessages").orderBy("createdAt");
+
+export const createMessage = (firestore, user, message) =>
+  firestore()
+    .collection("generalMessages")
+    .add({
+      message,
+      createdAt: new Date().getTime(),
```

Comment:
I think right now this is something you don't have to worry but keep in mind in the future if the chat it's available also in web maybe you can have some issues about reading the correct value in any device so I think it's a good idea to have a parser util that manage/transform the date to the same format for any platform to have consistency across multiple apps